bryan/MongooseDashboard_Modularized/client/views/edit.ejs

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/css/styles.css">
+  <title>Animals Database</title>
+</head>
+
+<body>
+
+  <div id="container">
+    <h1>Animals Database</h1><br /><br />
+    
+    <h2>Editing: <span class="animal"><%= name %></span></h2> <br />
+
+    <form action="/animals/update/<%= animal._id %>" method="post">
+
+      <label for="name">Name:</label>
```

Comment:
FYI, when you're adding the 'for' attribute to the label tag, you need to add an 'id' attribute to the corresponding input tag with the same name, otherwise the 'for' doesn't actually do anything.